### PR TITLE
Allow Patron users to disable daily DMs

### PR DESCRIPTION
**Pull request by TastyPumPum**
Allows users to disable daily dms if they so wish.

- [ ] I have tested all my changes thoroughly.

### CODE REVIEW

**Comment by TURB042O** [comment]
_Future Proposal posted by @mylife212 in https://github.com/oldschoolgg/oldschoolbot/issues/5263

File: src/lib/tickers.ts [hunk 1]
```diff
@@ -142,19 +142,27 @@ export const tickers: { name: string; interval: number; timer: NodeJS.Timeout |
...
+					','
+				)}}'::int[] AND user_stats."last_daily_timestamp" != -1 AND to_timestamp(user_stats."last_daily_timestamp" / 1000) < now() - INTERVAL '${reminderHours} hours';
+                `
 			);
 
-			for (const row of result.values()) {
+			for (const row of result) {
 				if (!production) continue;
 				if (Number(row.last_daily_timestamp) === -1) continue;
+				if (row.bitfield === 31) continue;
```
Comment (gc) [inline src/lib/tickers.ts:165]: `if (row.bitfield === 31) continue;` i dont really understand this, isnt bitfield an array and this would throw an error?

**Review by gc — CHANGES_REQUESTED** [review]

**Comment by gc** [comment]
Did you manually test the sql?

**Comment by TastyPumPum** [comment]
> Did you manually test the sql?

No I'm not too sure to test really, that's why I've left the label there for now, if you can assist with testing or want to take over this PR feel free

**Pull request closed without merging**